src/ShallowWrapper.js

```diff
@@ -80,8 +80,16 @@ export default class ShallowWrapper {
           }
         });
       });
-      this.node = this.renderer.getRenderOutput();
-      this.nodes = [this.node];
+      Object.defineProperty(this, 'node', {
+        get() {
```

Comment:
While I agree that getters don't improve readability, this is definitely a bug. Ask anyone with passing familiarity with Enzyme about what this should do and compare it with what actually happens:

```javascript
let wrapper = Enzyme.shallow(<MyInput debounce={ 500 } />)
wrapper.simulate("keypress", {char: "b"})
wrapper.simulate("keypress", {char: "a"})
wrapper.simulate("keypress", {char: "r"})
expect(wrapper.find(".result").text()).toBe("")
jasmine.clock().tick(1000)
expect(wrapper.find(".result").text()).toBe("bar")
```